README.md

```diff
@@ -1 +1,39 @@
-# codice-parent
\ No newline at end of file
+<!--
+/*
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version. 
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+-->
+
+<img src="http://www.codice.org/images/codice-logo.png"/>
+
+# [Codice Parent \(codice-parent\)](http://www.codice.org)
+[![Build Status](https://api.travis-ci.org/codice/codice-parent.png)](https://travis-ci.org/codice/codice-parent?branch=master)
+
+
+Codice-Parent is an open source project incorporated within the DDF integration framework. 
```

Comment:
❓ Should we leave references to DDF in a Codice parent?  I was not expecting to see this in this repo.